src/pages/trader-tools/margin-calculator/_margin-calculator.tsx

```diff
@@ -335,7 +335,7 @@ const MarginCalculator = () => {
                                     <FormulaContainer height="185px">
                                         <FormulaHighlight>
                                             <FormulaValue>
-                                                <Localize translate_text="2" />
+                                                <Localize translate_text="(2" />
```

Comment:
I don't think we need to localize these numbers. @fiona-deriv Was there a specific requirement for it?